### ISSUE TITLE
`isProto()` returns `false` even though the node is PROTO

### ISSUE THREAD

**Issue by lukicdarkoo** [cyberbotics/webots#2392]
**Describe the Bug**
`supervisor.getFromDef('ROBOT_A').isProto()` returns `false` if called from ROBOT_A even though ROBOT_A is PROTO. If the same snippet is called from another robot it returns `true`.

**Steps to Reproduce**
1. Download [get_from_proto_def.zip](https://github.com/cyberbotics/webots/files/5440255/get_from_proto_def.zip)
2. Run the world `get_from_proto_def.wbt`

**Expected output**
```
isProto True
getFromDef None
getFromProtoDef <controller.Node; proxy of <Swig Object of type 'webots::Node *' at 0x000002D3A581A4E0> >
```

**Actual output**
```
Error: wb_supervisor_node_get_from_proto_def(): 'node' is not a PROTO node.
isProto False
getFromDef None
getFromProtoDef None
```

**Comment by lukicdarkoo** [comment]
This seems to be regression, it works fine in R2020b (originally tested with R2021a)

**Comment by lukicdarkoo** [comment]
Closed by #2395
